easybuild/easyconfigs/j/Jorg/Jorg-1.0.0-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+# Author: Ben Langenberg ben.langenberg@ufz.de
+# Helmholtz-Centre for Environmental Research
+# Leipzig - Germany
+
+easyblock = 'PackedBinary'
+
+name = 'Jorg'
+version = '1.0.0'
+
+homepage = 'https://github.com/lmlui/Jorg'
+description = "A MAG Circularization Method By Lauren Lui, Torben Nielsen, and Adam Arkin"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+github_account = 'Helmholtz-UFZ'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['74b1c455ba57ab240921966255ca059840b97c47efcd16e9e66b9b6a8a136ee2']
+
+
+prepend_to_path = ['.']
+
+dependencies = [
+    ('MIRA', '5.0rc2'),
+    ('seqtk', '1.3'),
+]
+
+sanity_check_paths = {
+    'files': ['jorg'],
+    'dirs': [],
+}
+
```

Comment:
can you add a `sanity_check_commands` here? for example (I haven't tested this)
```
sanity_check_commands = ['jorg --help']
```

Reply:
yes, thank you

Reply:
yes I' gone the harder way. I've patched jorg to exiting 0 in every help sequence. This is why the tag here changed to 1.0.1. Your suggest workarround seems to be much easier :)